ParisBNF/abb/BNFabb103.xml

```diff
@@ -0,0 +1,306 @@
+                                    reason="owner" unit="chars" quantity="7"/> ለዓለመ፡ ዓለም፡ አሜን። ገድለ፡
+                                ኂሩቱ፡ ለብፁዕ፡ ብእሴ፡ እግዚአብሔር፡ ዝውእቱ፡ ገብረ፡ ክርስቶስ፡ ወልደ፡ ቴዎደስዮስ፡ ንጉሠ፡ ቍስጥንጥያ።
+                                ወቴዎዶስዮስሰ፡ መፍቀሬ፡ እግዚአብሔር፡ ውእቱ፡ ወመፍቀሬ፡ እንግዳ፡</incipit>
+                            <explicit xml:lang="gez">ወወረደ፡ ብርሃን፡ ኀበ፡ መቃብሩ፡ ዘየሐዩ፡ ኵሎ፡ ድዉያነ፡
+                                ወሕሙማነ።</explicit>
+                            <explicit xml:lang="gez" type="supplication">ጸሎቱ፡ ወበረከቱ፡ ወትጋሃ፡ ጸማሁ፡
+                                ለብእሴ፡ እግዚአብሔር፡ <hi rend="rubric">ገብረ፡ ክርስቶስ፡</hi> የሀሉ፡ ምስለ፡ አግብርቱ፡
+                                    <space reason="owner" unit="chars" quantity="15"/>
+                                <del rend="effaced" unit="chars" quantity="10"/>
+                                <space reason="owner" unit="chars" quantity="15"/> ለዓለመ፡ ዓለም፡ አሜን።
+                                ወአረፈ፡ አመ፡ ፲ወ፬ በወርኃ፡ ጥቅምት፡ በዕለተ፡ እሑድ፡ ጊዜ፡ ፫ሰዓት፡ ለዘጸሐፎ፡ ወለዘአጽሐፎ፡
+                                ወለዘአንበቦ፡ ወለዘሰምዓ፡ ቃላቲሁ፡ ኅቡረ፡ ይምሐረነ፡ እግዚአብሔር፡ አሜን።</explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="28ra" to="50ra"/>
+                            <textLang mainLang="gez"/>
+                            <title type="complete" ref="LIT1959Mashaf" xml:lang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubrication">በስመ፡ አብ፡ ወወልድ፡</hi>
+                                ወመንፈስ፡ ቅዱስ፡ ፩አም<hi rend="rubric">ላከ፡ ቀኖና፡ መጽሐፈ፡ ቄ</hi>ድር፡ ዘአንበሩ፡
+                                    መምህራ<hi rend="rubrication">ን፡ በእንተ፡ ዘክህደ፡ ሃይ</hi>ማኖቶ። አው፡ ዘአርኰሰ፡
+                                ሥጋሁ፡ ምስለ፡ ኢምእመናን። አው፡ ተባዕተ፡ አው፡ አንስተ።</incipit>
+                            <explicit xml:lang="gez">ወዘንተ፡ ሕዝብ፡ ዘዐለወ፡ ጽኑሕ፡ ሎቱ፡ ደይን፡ ወበዛቲ፡ ሕግ፡ ዘሖረ፡
+                                ድኁን፡ እምኵነኔ፡ እወ፡ አሜን። ወአሜን፡ </explicit>
+                            <explicit xml:lang="gez" type="supplication">ከማሁ፡ ያድኅኖሙ፡ ለአግብርተ፡
+                                እግዚአብሔር፡ <hi rend="rubric"><persName ref="PRS14837HadaraK">ሐደራ፡
+                                        ክርስቶስ፡</persName>
+                                    <persName ref="PRS14833Zawangel">ዘወንጌል።</persName></hi> ወለአእማተ፡
+                                እግዚአብሔር፡ <hi rend="rubric"><persName ref="PRS14836SablaW">ሰብለ፡ ወንጌል፡ </persName>
+                                    <persName ref="PRS14835AdamS">አዳም፡ ስና፡</persName></hi>
+                                <space reason="rubrication"/> እስከ፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን። </explicit>
+                            <note><bibl>
+                                    <ptr target="bm:Chaine1912Catalogue"/>
+                                    <citedRange unit="page">65</citedRange>
+                                    <citedRange unit="number">103</citedRange>
+                                </bibl> distinguishes ቀኖና፡ ደግመ፡ ክርስትና። on f. 34v. </note>
```

Comment:
The Geez text should be marked up with foreign, and it is better to use locus for folio indications for the sake of consistency. Not knowing this work well, I must say that I do not understand this note. From what does he distinguish it? Maybe "indicates that... is located on..."?

Reply:
> The Geez text should be marked up with foreign, and it is better to use locus for folio indications for the sake of consistency. Not knowing this work well, I must say that I do not understand this note. From what does he distinguish it? Maybe "indicates that... is located on..."?

I will just delete it :-) His note is unclear anyways.